.github/workflows/release-app.yml

```diff
@@ -51,18 +51,18 @@ jobs:
       matrix:
         include:
```

Comment:
does this run?

last time someone sent a PR on this I tested and it was crashing (wasted my time)

Reply:
It did run for me yes